lib/tasks/temp.rake

```diff
@@ -323,4 +323,9 @@ namespace :temp do
   task :set_use_notifications => :environment do
     InfoRequest.update_all use_notifications: false
   end
+
+  desc 'Set a default time for users daily summary notifications'
+  task :set_daily_summary_times => :environment do
+    User.update_all daily_summary_hour: 8, daily_summary_minute: 30
```

Comment:
Hmm, good point! I've added some initialisation in dde11e3 copying what we already do for alert timings (setting a random time). I've also added 87673b5 to make the rake task work like this too, so that it sets a random hour and minute for each user. However, to make this fast I did it in SQL which I've now realised is Postgres-specific. Let me know if this is an issue.